packages/v4/src/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -72,6 +72,21 @@ Sub-steps can be added to the sidebar. Use sub-steps when there is a hierarchica
 
 * Parent steps are automatically closed once they are complete. However, the user can manually expand the parent back open at any time to see the sub-steps again while staying in context of their current step.
 
+### Wizard with optional steps
+Optional steps can be added to the sidebar. Use optional steps when there are steps that are do not need to be filled out to complete the wizard.
```

Comment:
I would just say "Optional steps can be added to the wizard". Might be unclear what the "sidebar" is!

Also you have a typo in that second sentence - you have an extra "are" in there! I would also just reword to say "Make steps optional when they do not need to be filled out to complete the wizard."